initial suggestions for ReadMe

christinerose opened the pull request:
These are some suggestions for things I came across when trying out the tutorial as someone new to OCaml. I suggested changing `opam` to opam because Thomas said he preferred that formatting, and we're striving to make documentation consistent. 

I'll keep going through the tutorial if this is helpful, but I didn't want to put too much in one PR.

kayceesrk commented on README.md:
```diff
@@ -1,34 +1,42 @@
 
 Since we will be doing performance measurements, it is recommended that you also
-install [`hyperfine`](https://github.com/sharkdp/hyperfine).
+install [`hyperfine`](https://github.com/sharkdp/hyperfine). With Homebrew, `brew install hyperfine`
```
Homebrew is macOS only. Is there a specific reason only to include homebrew instructions here? `hyperfine` README includes installation instructions for a variety of platforms. I would suggest that we drop this addition.

christinerose replied:
Sure, let's just drop it.

kayceesrk commented:
Thanks :-) I've left some minor comments.

christinerose commented on README.md:
````diff
@@ -53,14 +61,18 @@ I ran in parallel
 ```
 
-The dune command compiles the native version of the above program and runs it:
+The Dune command compiles the native version of the above program and runs it:
````
Great! If it's the command, we generally wrap it in monospace. Would `dune` work?

christinerose commented:
Hi @kayceesrk! I addressed your comments, so please let me know if there's anything else I need to do for this initial PR. I'll continue with the rest of the tutorial in a separate PR.

kayceesrk commented on README.md:
```diff
@@ -1,34 +1,42 @@
 
 A hands-on tutorial on the new parallelism features in OCaml 5. This tutorial
-was run on 19th May 2022.
+was run on 5 July 2022. Currently, the alpha version of OCaml 5 has been released, and the full version is set for release in September 2022. 
```
We ran the tutorial on 19th May 2022 at the Tarides offsite. I didn't run it on 5th July 2022. 

I wanted to have a date in the README since it is very likely that the tutorial will go out of date with all the breaking changes that are happening as we head towards the release of OCaml 5.

Perhaps we can make it clear by rewording the sentence as "was run on the 19th of May 2022 at the [Tarides retreat](https://tarides.com/blog/2022-06-23-team-tarides-visits-a-17th-century-chateau)".

christinerose replied:
Revised! 
I put that in since I was running it, but it's best to keep it when you ran it fully.

kayceesrk commented:
I've left a comment about the date. Otherwise, it looks good.
